src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,55 @@
+package model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public Operation getOperation() {
```

Comment:
The method `getOperation()` should return a `String` instead of `Operation`. This way, the `Operation` enum is encapsulated within the `Transaction` class and is not exposed to other classes.